content/_guides/creating-recipes.md

```diff
@@ -5,166 +5,87 @@ section: Recipes
-                "config": {
-                  "mapper_type": "jsonata",
-                  "mapper": {
-                    "to": "pets[0].name",
-                    "cc": "$getFlowVariables().cc",
-                    "subject": "pets[0].id",
-                    "textBody": "pets[0].status"
-                  },
-                  "condition": null
-                },
-                "source": "step_1",
-                "target": "step_2"
-              }
-            ]
-          }
-        }
-      },
-      "relationships": {
-        "workspace": {
-          "data": {
-            "type": "workspace",
-            "id": "{WORKSPACE_ID}"
-          }
-        }
-      }
-    }
-  }'
+   {
+     "data": {
+       "visibility": "contract"
+     }
+   }'
 ```
 
-### Creating Recipes from Scratch
+> The **"visibility"** field can take the following values: **"workspace"**, **"contract"** or **"tenant"**.
 
-To create a Recipe from a chosen Flow, use the following request, provided you have `workspaces.recipe.edit` permission:
+After changing the visibility to **contract**, the recipe setup will look like this:
+
+![Contract visibility](/assets/img/integrator-guide/creating-recipes/contract-visibility.png)
+
+And after changing the visibility to **tenant**, the recipe setup will look like this:
+
+![Tenant visibility](/assets/img/integrator-guide/creating-recipes/tenant-visibility.png)
+
+  **3.** Here you can setup your credentials for every step. You can verify(**1**), edit(**2**) or delete(**3**) credentials and choose credentials you need(**4**):
+
+![Setup recipe credential](/assets/img/integrator-guide/creating-recipes/recipe-cred-setup.png)
+
+  **4.** After that you have to click to "Finish" and new identical flow with the same name will be created:
+
+![New flow](/assets/img/integrator-guide/creating-recipes/new-flow.png)
+
+## Creating Recipes using API
+
+There are two ways to create a Recipe:
 ​
-`POST {{site.data.tenant.apiBaseUri}}/v2/flows/{FLOW_ID}/export-to-recipe`​
+- [From an existing integration Flow](#creating-recipes-from-flows)
+​
+- [From scratch](#creating-recipes-from-scratch)
 
+### Creating Recipes from Flows
 
-Below are the request body parameters:
+To create a Recipe from a chosen Flow, use the following request, provided you have `workspaces.recipe.edit` permission:
 ​
 
-| **Parameter** | **Required** | **Description** |
-| `type`                              | yes          | Allowed value: `recipe`    |
-| `relationships.workspace.data.id`   | yes          | Workspace ID               |
-| `relationships.workspace.data.type` | yes          | Allowed value: `workspace` |
+`POST /v2/recipes/`
 
+To see parameters and examples please visite [API-Documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#create-a-recipe-from-existing-flow).
 
-**EXAMPLE:**
+### Creating Recipes from Scratch
+
+To create a Recipe from a chosen Flow, use the following request, provided you have `workspaces.recipe.edit` permission:
```

Comment:
sorry, from scratch or from a chosen flow ??? it looks, like the same, as the section above

Reply:
corrected, done